pkg/reconciler/route/traffic/traffic.go

```diff
@@ -78,17 +77,9 @@ func BuildTrafficConfiguration(configLister listers.ConfigurationLister, revList
 	return builder.build()
 }
 
-// GetDomains returns all of the domains and tag domains (sub-domains) of a Route
-func GetDomains(domain string, targets map[string]RevisionTargets) []string {
```

Comment:
The original `GetDomains` uses the `TagDomain` function, which is deprecated after `Kevin` is introduced.